### DISCUSSION TITLE
WIMUtil General Discussion

### DISCUSSION THREAD

**Discussion by memstechtips** [memstechtips/WIMUtil#3] [Q&A]
Use this discussion page for general topics related to WIMUtil.
Please don't submit any issues here, use [Issues](https://github.com/memstechtips/WIMUtil/issues) for that.

**Comment by Maxarian** [comment]
Cannot complete Step 5 as receiving message 'Signature date mismatch! oscdimg may not be from Microsoft.'
![image](https://github.com/user-attachments/assets/a3c8d375-4b14-4b05-820b-3fe84a98c4e1)
